packages/components/src/LoadingOverlay.scss

```diff
@@ -13,9 +13,14 @@ $iris-panel-scrim-bg: rgba(black, 0.1);
   overflow: hidden;
 
   .message-content {
+    display: flex;
+    flex-direction: column;
     font-size: $iris-panel-message-font-size;
+    gap: 20px;
     .message-icon {
       font-size: $iris-panel-icon-font-size;
+      height: $iris-panel-icon-font-size;
```

Comment:
I set height + line height explicitly to avoid a layout shift after font load. The previous height was 96px based on 1.5 line-height from body. Converting to flexbox with a gap of 20px seems to match the layout as before but without the layout shift